setup.py

```diff
@@ -0,0 +1,28 @@
+from distutils.core import setup
+setup(
+  name = 'swmr_tools',         # How you named your package folder (MyLib)
+  packages = ['swmr_tools'],   # Chose the same as "name"
+  version = '0.3',      # Start with a small number and increase it with every change you make
+  license='MIT',        # Chose a license from here: https://help.github.com/articles/licensing-a-repository
+  description = 'Python iterator for safely monitoring nexus files',   # Give a short description about your library
```

Comment:
NeXus or should it be HDF5?

Reply:
Changed just to NeXus. In theory it should work for any hdf5 file setup in a similar way, but it was specifically built for NeXus.